worker/build.sbt

```diff
@@ -6,56 +6,32 @@ version := "1.11"
 
 scalaVersion := "2.11.7"
 
-crossScalaVersions := Seq(scalaVersion.value, "2.11.6")
+crossScalaVersions += "2.10.5"
```

Comment:
let's just take this out - we don't need it and nobody else is using piezo now I believe